Sample_Anuglar2/.gitattributes

```diff
@@ -0,0 +1,63 @@
+###############################################################################
+# Set default behavior to automatically normalize line endings.
```

Comment:
ok so i will delete this on and make changes in the root folder file with this folder path